lib/application/auth/username_bloc.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:async';
+
+import 'package:bloc/bloc.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:injectable/injectable.dart';
+
+import '../../domain/auth/auth_failures.dart';
+import '../../domain/auth/i_auth_repository.dart';
+
+part 'username_event.dart';
+part 'username_state.dart';
+part 'username_bloc.freezed.dart';
+
+@injectable
+class UsernameBloc extends Bloc<UsernameEvent, UsernameState> {
+  final IAuthRepository profileRepository;
```

Comment:
Why IAuthRepository? It should be `IProfileRepository`?

Reply:
After changing to IProfileRepository Next button on enter_username.dart is not working, not getting any error also.
No errors getting, so that's why not sure what's not working.

Reply:
Should I push code to review?